reana_client/validation/environments.py

```diff
@@ -43,6 +43,11 @@ def build_validator(workflow):
         if workflow_type == "cwl":
             workflow_file = workflow.get("file")
             return CWLEnvironmentValidator(workflow_file=workflow_file, pull=pull)
+        if workflow_type == "snakemake":
+            workflow_steps = workflow.rules()
```

Comment:
This statement fails for me with `AttributeError: 'dict' object has no attribute 'rules'`. `workflow` is a dict that has something like: 

```python
{
  'file': 'workflow/snakemake/Snakefile'
  'specification': {}
  'type': 'snakemake'
}
```
so it's not possible to call `rules` like that.

The idea here would be to load the Snakefile ( path in `workflow['file']) using some `snakemake` parsing function to later grab the multiple rules. Maybe there's some code to perform this load that you haven't pushed?